src/poliastro/plotting/groundtrack.py

```diff
@@ -0,0 +1,143 @@
+""" This script holds the function for plotting the groundtracks. """
+import numpy as np
+import plotly.graph_objects as go
+from astropy import units as u
+from astropy.coordinates import (
+    GCRS,
+    ITRS,
+    CartesianRepresentation,
+    SphericalRepresentation,
+)
+from astropy.time import TimeDelta
+
+from poliastro.examples import iss
+from poliastro.twobody import propagation
+
+
+def groundtrack(
```

Comment:
This huge amount of function parameters can be avoided while keeping the same customization degree. Many of those variables end up being the keys of a dictionary one (which is the idea behind all plotly plotters). Check out below comments for a deep explanation.

Furthermore, in the original issue:

* What about if a user wants to know where was a satellite some time ago? Is it possible to pass a negative time of flight value and get the old groundtrack?